### PR TITLE
Added target for openlane flow

### PR DESCRIPTION
**Pull request by microSharjeel**
Hi Sir,
This PR has changes for target for OpenLane flow. This target first starts the Docker container and then runs the flow for asic soc. 
Things to be addressed next include,
1. Inclusion of this Makefile into asic.mk .... hardware.mk.....system.mk flow so that system.mk should be able to call this Makefile. 
2. In this Makefile I am using example design from openlane for test only. For iob-soc I need to pass the VSRC and VHDR variables to the flow.tcl
regards

### CODE REVIEW

File: hardware/asic/README.md [hunk 1]
```diff
@@ -28,8 +28,14 @@ where library name is one of the following:
 3.  sky130_fd_sc_ms
 4.  sky130_fd_sc_ls
 5.  sky130_fd_sc_hdll
-
-
+In order to run OpenLane from Makefile provided in the hardware/asic/skywater folder, you need to export following environment variable.
+```bash
+export OPENLANE_DIRECTORY=<path to OpenLane root directory>
```
Comment (jjts) [inline hardware/asic/README.md:33]: I suggest OPENLANE_HOME

File: hardware/asic/README.md [hunk 2]
```diff
@@ -62,9 +61,16 @@ sudo usermod -aG docker $USER
 ```bash
 newgrp docker
```
Comment (jjts) [inline hardware/asic/README.md:62]: I don't get this: the group has been created already. Isn't it created when you install docker?

File: hardware/asic/README.md [hunk 3]
```diff
@@ -62,9 +61,16 @@ sudo usermod -aG docker $USER
...
+## OPENLANE FLOW FOR MACRO HARDENING FROM AN HDL DESIGN
+
+1. Go to the OpenLane installation directory (you may set an environment variable called OPENLANE_HOME that points to this directory) and start the OpenLane Docker container by running 
+
+```bash
+make mount
+```
+The "make mount" command opens up a bash terminal in which you run the following tcl script to generate a design configuration of your design:
 ```bash
 flow.tcl -design design_name -init_design_config
```
Comment (jjts) [inline hardware/asic/README.md:75]: show the bash prompt here

File: hardware/asic/skywater/Makefile [hunk 4]
```diff
@@ -1,19 +1,26 @@
...
-all:  Dok 
-Dok:
-	$(MAKE) -C  $(OPENLANE_MK)  -f Makefile mount 
-
-.PHONY: all Dok 
 
+#soc flow for OpenLane
+soc:
+	cd $(OPENLANE_DIRECTORY) && \
+		$(DOK_CMD) sh -c  "./flow.tcl -design APU  --init_design_config -overwrite"
```
Comment (jjts) [inline hardware/asic/skywater/Makefile:19]: is this an alternative to moving into the directory ?  Why have both?

File: hardware/asic/README.md [hunk 5]
```diff
@@ -62,9 +61,16 @@ sudo usermod -aG docker $USER
...
 
-This command opens up a bash terminal in which you run the following tcl script to generate a design configuration of your design:
+## OPENLANE FLOW FOR MACRO HARDENING FROM AN HDL DESIGN
+
+1. Go to the OpenLane installation directory (you may set an environment variable called OPENLANE_HOME that points to this directory) and start the OpenLane Docker container by running 
+
+```bash
+make mount
+```
+The "make mount" command opens up a bash terminal in which you run the following tcl script to generate a design configuration of your design:
```
Comment (jjts) [inline hardware/asic/README.md:73]: In which line in the README the user is instructed to create the design files? Where should the user put them?

File: hardware/asic/README.md (hunk 2 above)
Comment (microSharjeel) [reply hardware/asic/README.md:62]: No sir the make in openlane will not do this .......... I was stuck at this point ......... it is important

File: hardware/asic/README.md (hunk 5 above)
Comment (microSharjeel) [reply hardware/asic/README.md:73]: I have provided the required explanation.

File: hardware/asic/README.md (hunk 3 above)
Comment (microSharjeel) [reply hardware/asic/README.md]: Added

File: hardware/asic/README.md (hunk 1 above)
Comment (microSharjeel) [reply hardware/asic/README.md]: Changed accordingly

File: hardware/asic/skywater/Makefile (hunk 4 above)
Comment (microSharjeel) [reply hardware/asic/skywater/Makefile:19]: Right now it is ok as u said in hangout conversation

File: hardware/asic/README.md (hunk 5 above)
Comment (microSharjeel) [reply hardware/asic/README.md]: Kindly check changes

File: hardware/asic/skywater/Makefile [hunk 6]
```diff
@@ -1,19 +1,26 @@
 #Makefile for OpenLane and OpenRAM started on 2/11/21
```
Comment (jjts) [inline hardware/asic/skywater/Makefile:2]: ROOT_DIR=../../..

need this so your makefile can find root

**Comment by jjts** [comment]
also add debug target to display variables

**Pull request merged**